packages/grafana-ui/src/components/Icon/utils.ts

```diff
@@ -43,3 +43,20 @@ export function getSvgSize(size: IconSize) {
       return 48;
   }
 }
+
+let iconRoot: string | undefined;
+
+export function getIconRoot(): string {
```

Comment:
Trivial, but probably would still worth adding a test for this new function in `utils.test.ts`.